pkg/virt-handler/node-labeller/api/capabilities.go

```diff
@@ -100,3 +104,17 @@ func (b *yesnobool) UnmarshalXMLAttr(attr xml.Attr) error {
 	}
 	return fmt.Errorf("value %v of %v is not (yes|no)", attr.Value, attr.Name)
 }
+
+func (b *CPUSiblings) UnmarshalXMLAttr(attr xml.Attr) error {
+	if attr.Value != "" {
+		if list, err := hwutil.ParseCPUSetLine(attr.Value, 100); err == nil {
```

Comment:
Can't this easily be more than 100?
I think the limit is much higher, around 8192

Reply:
100 is just the limit for cpu threads. For `GetPodCPUSet` where we are reading what we assign to the pods it is set to `50000`.

Reply:
ah, I see.